atlasdb-container-test-utils/src/test/java/com/palantir/atlasdb/containers/CassandraCliParserTest.java

```diff
@@ -18,10 +18,27 @@
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
 
+import org.junit.Before;
 import org.junit.Test;
 
 public class CassandraCliParserTest {
-    public static final String CORRUPT_STRING = "sodu89sydihusd:KSDNLSA";
+    private static final String CORRUPT_STRING = "sodu89sydihusd:KSDNLSA";
+    private CassandraCliParser parser;
+
+    @Before
+    public void setUp() throws Exception {
+        parser = new CassandraCliParser(CassandraVersion.from("2.2.8"));
```

Comment:
This should just be: `private final CassandraCliParser parser = new CassandraCliParser(CassandraVersion.from("2.2.8"));`